src/components/List/ListOptimizer/ListOptimizer.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { List, CellMeasurer, CellMeasurerCache, AutoSizer, ListRowProps } from 'react-virtualized';
+
+export interface Props {
+    rowCount: number;
+    defaultHeight?: number;
+    defaultWidth?: number;
+    children: (props: ListRowProps) => React.ReactNode;
+}
+
+const DEFAULT_HEIGHT = 30;
```

Comment:
maybe it makes sense to rename this constant to `DEFAULT_CELL_HEIGHT` or something? Current naming in combination with `defaultHeight` prop of the component below makes it a bit ambiguous.

Reply:
Yep, good point

Reply:
Good point